### DISCUSSION TITLE
v0.10.9: Unstable Release

### DISCUSSION THREAD

**Discussion by Saugia** [endless-sky/endless-sky#10515] [General]
This is an unstable release, introducing new content and features that may contain bugs. Major new changes and additions include the following:

* Boarding now pits attack against defense when both the attacker and defender choose the attack option.
* A new fleet formation mechanic has been added with several different formations the player can cycle through.
* A fleet "hold fire" command has been added.
* Two new alien species have been added to the game - the young Incipias north of Hai space, and the elusive Successors south of The Graveyard.

Some other notable additions and changes to content and mechanics are:
* Wanderer planets now have defense fleets and can be demanded tribute.
* The Dragonflame Cannon now pierces through its target.
* Several new missions involving the Gegno.
* New scroll bar for various UI elements.

For a complete list of changes, as well as further detail about the summarized changes, see the [changelog](https://github.com/endless-sky/endless-sky/blob/master/changelog). Special thanks to everyone who contributed to this release: @alexrovw @Amazinite @Arachi-Lover @aradapilot @AvianGeneticist @Azure3141 @bene-dictator @blue-four-seven @Daeridanii1 @eebop @Ebreus @GefullteTaubenbrust2 @Hurleveur @implicitfield @Koranir @mOctave @moonliiiqhte @nickshanks @petervdmeer @Quantumshark @RisingLeaf @reticent-rem @roadrunner56 @Saugia @SomeTroglodyte @stefanct @Tangle10 @tekker2234 @tibetiroka @TheGiraffe3 @thewierdnut @TomGoodIdea @UnorderedSigh @warp-core @ziproot

If you are using the high-dpi plugin, you should download and install the [updated version](https://github.com/endless-sky/endless-sky-high-dpi/releases/tag/v0.10.9) that goes with this release.

![emeraldsword_800x450](https://github.com/user-attachments/assets/1ea9575d-4b97-45bd-a1ae-478e93cab741)

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/endless-sky/releases/tag/v0.10.9'>v0.10.9: Unstable Release</a>.</em>

**Comment by SomeTroglodyte** [comment]
<details><summary>Minor minor very minor issue:</summary>

![image](https://github.com/user-attachments/assets/72b2220f-c756-4627-a406-421ee5613cb4)
... mission "Successors: New Houses Rush Delivery" fails to consider the time required to depart and land again. Or it shouldn't allow source and destination (system) to be equal.

Also, if one lands on source for "Successors: Researchers Out 1" before going to destination the message says you left your escorts behind - untrue in my case.
</details>

**Reply by Saugia** [reply]
Thanks! Will look at getting to this in this next release; it's what the unstable -> stable period is for. :D

**Reply by warp-core** [reply]
Clearly, the job is intended to be completed by reloading the save instead of launching.

**Reply by SomeTroglodyte** [reply]
> reloading the save instead of launching

Something which shouldn't even be possible but is... Also showcases that it should be possible to write missions that can be completed right away without taking off - but we can't. Also - the twin to the fail action is still missing... Also - I was reporting a quirk in a suboptimal place, but I just recovered from a hours-long OS repair which made me lazy...[^1]

[^1]: <sub><sup>If you have Mint 22 and your system encrypted on a LUKS partiton, **do not** update your kernel to 6.8.0-45.45 - it breaks the passphrase entry. And if you try boot-repair on that, that tool destroys grub entirely. Only timeshift always works...</sup></sub>

**Comment by TheGiraffe3** [comment]
Who forgot to close the milestone, that now says it's past due by one day and yet the release has been made?..
👀

**Comment by TheGiraffe3** [comment]
> and can be demanded tribute

Should be "and can have tribute demanded".
